src/util.ts

```diff
@@ -0,0 +1,8 @@
+// access any environment variable with env('KEY')
+export const env = (value: string): string | undefined => {
```

Comment:
File can be moved to tools folder and add a section in README.md telling how to use this ;)

Reply:
Done!